src/main/java/study/step3/Application.java

```diff
@@ -0,0 +1,19 @@
+package study.step3;
+
+import study.step3.domain.Cars;
+import study.step3.view.InputView;
+import study.step3.view.ResultView;
+
+public class Application {
+    public static void main(String[] args) {
+        final int numberOfCars = InputView.numberOfCars();
+        final int repeatCount = InputView.repeatCount();
+
+        final Cars cars = new Cars(numberOfCars);
+
+        for (int i = 0; i < repeatCount; i++) {
```

Comment:
```suggestion
        while(cars.isEndGame()) {
        }
```
위와 같이 Cars에 메시지를 보내 게임 종료 여부를 판단하도록 구현하는 것은 어떨까?
도메인 객체에 메시지를 보내기 위해 노력해 본다.